srcpkgs/inxi/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'inxi'
 pkgname=inxi
-version=3.3.25
+version=3.3.26
```

Comment:
Wouldn't it be better to include the `-1` in version, since that is what the upstream is using?

Reply:
Hyphens are not allowed in `version`, but you can instead do something like this:
```suggestion
version=3.3.26.1
_distver="${version%.*}-${version##*.}"
```
and then use `${_distver}` instead of `${version}` in `distfiles`.

Reply:
Thanks, that looks good.

Reply:
You forgot to update the commit message and PR title. :)

Reply:
Oh yea, sorry :D

I hope this format is correct, meaning exactly what's in `version`, so `3.3.26.1`, not `3.3.26-1`.

Reply:
It is, yeah.